python3-aeneas.yml

```diff
@@ -0,0 +1,49 @@
+# NOTES:
+  # 1.7.3.0 is the latest aeneas version on pypi. Are there newer versions
+  # and sources elsewhere? 
+  # This file uses version 1.25.2 of numpy which worked without having to
+  # install extra build modules. The latest version of numpy is 2.0.0 which
+  # will require 'meson-python' to build successfully.
+  # A setup error in version 1.17.5 of numpy suggested the "setup.py clean"
+  # function was deprecated for builds. This is why I was testing new versions.
+  # FYI: The last version of numpy released as a .zip instead of a .tar.gz
+  # was 1.22.4: https://pypi.org/project/numpy/1.22.4/#files
+name: python3-aeneas
+buildsystem: simple
+build-commands:
+    - python3 -m pip install --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "numpy" --no-build-isolation
+    - python3 -m pip install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "aeneas" --no-build-isolation
+sources:
+    - type: file
+      url: https://files.pythonhosted.org/packages/e5/92/88d421001bb257588df4864ceca24d570e2e822db4f40f48737a78b648b2/aeneas-1.7.3.0.tar.gz
+      sha256: b492d8ba36f12cce69c29c35e7075857a9d036c4743cc8f4689922be980826da
+    - type: file
+      url: https://files.pythonhosted.org/packages/4c/f3/038b302fdfbe3be7da016777069f26ceefe11a681055ea1f7817546508e3/soupsieve-2.5-py3-none-any.whl
+      sha256: eaa337ff55a1579b6549dc679565eac1e3d000563bcb1c8ab0d0fefbc0c2cdc7
+    - type: file
+      url: https://files.pythonhosted.org/packages/63/f7/ffbb6d2eb67b80a45b8a0834baa5557a14a5ffce0979439e7cd7f0c4055b/lxml-5.2.2.tar.gz
+      sha256: bb2dc4898180bea79863d5487e5f9c7c34297414bad54bcd0f0852aee9cfdb87
+    - type: file
+      url: https://files.pythonhosted.org/packages/a0/41/8f53eff8e969dd8576ddfb45e7ed315407d27c7518ae49418be8ed532b07/numpy-1.25.2.tar.gz
+      sha256: fd608e19c8d7c55021dffd43bfe5492fab8cc105cc8986f813f8c3c048b38760
+    - type: file
+      url: https://files.pythonhosted.org/packages/b1/fe/e8c672695b37eecc5cbf43e1d0638d88d66ba3a44c4d321c796f4e59167f/beautifulsoup4-4.12.3-py3-none-any.whl
+      sha256: b80878c9f40111313e55da8ba20bdba06d8fa3969fc68304167741bbf9e082ed
+modules:
+    - name: espeak
+      # buildsystem: simple
+      # build-commands:
+      #     - |
+      #         set -xueo pipefail
+      #         cd src
+      #         make install
```

Comment:
```suggestion
```

Reply:
Sorry!  I commented it out since I wasn't sure it would work. and forgot to delete it.

Reply:
The whole file has been removed and the contents moved to the main manifest.  It does not contain the above commented out code.